src/createStudent.js

```diff
@@ -15,6 +15,12 @@
     estudante.feedback() // Retorna: 'Eita pessoa boa!'
 */
 
-const createStudent = () => {};
+const createStudent = (name) => {
+  let student = {
+    name,
+    feedback: () => 'Eita pessoa boa!',
+  };
+  return student;
+};
```

Comment:
O que acha de utilizar um return implicito, já que a sua função apenas retorna um objeto com os dados recebidos por parâmetro?
```suggestion
const createStudent = (name) => ({
    name,
    feedback: () => 'Eita pessoa boa!',
  });
```